.gitpod.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM gitpod/workspace-python-3.10 
+
+# VideoDataSet
+RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends libgl1
+RUN sudo apt-get install make
+RUN sudo apt-get install -y --no-install-recommends libatk-bridge2.0-0 libcups2 ca-certificates fonts-liberation libasound2 libatk-bridge2.0-0 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libglib2.0-0 libgtk-3-0 libnspr4 libnss3 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 lsb-release wget xdg-utils
```

Comment:
Several validation errors arise from https://www.fromlatest.io:

![image](https://github.com/kedro-org/kedro-plugins/assets/316517/c89d714a-3250-45c6-9a00-312f557db940)

even setting aside the `sudo` thing:

![image](https://github.com/kedro-org/kedro-plugins/assets/316517/3c7658b9-ccab-4b42-a610-66f1cbc40ecb)